### ISSUE TITLE
Refactor creation/read/update/delete of clauses in Induction

### ISSUE THREAD

**Issue by chrismear** [emmapersky/one-click-orgs#34]
Because at the moment, it hurts the eyes and the heart.

**Comment by chrismear** [comment]
Quite a bit already done in 610292b67759e570df62a24c33320523131f37a8.

**Comment by chrismear** [comment]
Use new clause getters in Induction#voting_settings. Closed by 3a56d8594df3bdfd18a212ae58064454492d3cf5.

**Comment by chrismear** [comment]
Closed for now, although it's possible that the lines like:

```
organisation_name = Clause.get_current('organisation_name') || Clause.new(:name => 'organisation_name')
```

could be replaced with something cleaner. However, I quite like keeping these explicit, because it makes it clear that we're routing around the usual clause setter behaviour and actually modifying the existing record.
